yarn.lock

```diff
@@ -3824,7 +3819,14 @@
     "@types/react" "*"
     "@types/reactcss" "*"
 
-"@types/react-dom@*", "@types/react-dom@^16.9.0":
+"@types/react-dom@>=16.9.0":
+  version "17.0.3"
```

Comment:
Hm, this is concerning, but it doesn't show up as a package to deduplicate

Reply:
It shows with the fewer strategy:

```
$ npx yarn-deduplicate --list --strategy fewer
Package "@types/react-dom" wants >=16.9.0 and could get 16.9.10, but got 17.0.3
...
```

The default strategy is implemented as: "for each version range, pick the highest version that can satisfy it from the installed versions". So it sees two versions, `16.9.0` and `17.0.3`, the former is the highest that can satisfy `^16.9.0` and the latter the highest that can satisfy `>=16.9.0`. 

'fewer' strategy is implemented as: "for each version range, find all installed packages that could satisfy it, find how many version ranges each one of those satisfies, and find use the package that satisfies the most". In this case it will see that `17.0.3` can satisfy one range and `16.9.0` can satisfy two ranges, therefore it will use `16.9.0` to solve `^16.9.0` and `>=16.9.0`.

I've pushed a fix with this package deduplicated.

Reply:
Awesome, thanks!